tests/lib/Service/OpenProjectAPIServiceTest.php

```diff
@@ -1237,6 +1237,110 @@ public function testGetOpenProjectAvatar() {
 		$this->assertSame('image/jpeg', $result['type']);
 	}
 
+	/**
+	 * @group ignoreWithPHP8.0
+	 * @return void
+	 */
+	public function testGetOpenProjectAvatarWithNoContentType() {
+		$consumerRequest = new ConsumerRequest();
+		$consumerRequest
+			->setMethod('GET')
+			->setPath('/api/v3/users/openProjectUserWithAvatar/avatar')
+			->setHeaders(["Authorization" => "Bearer 1234567890"]);
+
+		$providerResponse = new ProviderResponse();
+
+		$providerResponse
+			->setStatus(Http::STATUS_OK)
+			->setBody(null);
+
+		$this->builder
+			->uponReceiving('a request to get the avatar of a user that has an avatar')
+			->with($consumerRequest)
+			->willRespondWith($providerResponse);
+		$service = $this->getOpenProjectAPIService(null, '1234567890', 'https://nc.my-server.org', 'NCuser');
+		$result = $service->getOpenProjectAvatar(
+			'openProjectUserWithAvatar',
+			'Me',
+			'NCuser'
+		);
+		$this->assertArrayHasKey('avatar', $result);
+		//make sure its an image, if something else is returned it will throw an exception
```

Comment:
```suggestion
		// make sure its an image
```

It doesn't throw an exception. You could make the comment short, like this suggestion. The name of `assertNotFalse` already gives the clue about what is being checked.

Reply:
Okay